tsconfig.json

```diff
@@ -11,5 +11,11 @@
         }
     },
     "files": [".storybook/typings.d.ts"],
-    "include": ["src/**/*.ts", "src/**/*.tsx", "examples/src/**.tsx", "types.ts"]
+    "include": [
+        "src/**/*.ts",
+        "src/**/*.tsx",
+        "examples/src/**.tsx",
+        "types.ts",
+        "src/elements/content-sidebar/activity-feed/comment/components/BaseCommentMenu.js"
```

Comment:
What's the reason for including this file here?

Reply:
Ah, that's just IDE shenanigans. Thanks for catching that.